Dockerfile

```diff
@@ -30,6 +30,9 @@ COPY ./flink_jobs /opt/flink/py_libs/flink_jobs
 COPY ./dist/*.whl /opt/flink/py_libs/
 RUN mkdir /opt/flink/py_libs/scripts/
 COPY ./scripts/*.py /opt/flink/py_libs/scripts/
+COPY ./scripts/*.zip /opt/flink/py_libs/scripts/
+COPY ./scripts/*.sh /opt/flink/py_libs/scripts/
```

Comment:
Now he copies the data from this repo. I think it is worth keeping it here, as it can be also used manually, for testing, etc. If you do not agree let's discuss how it makes sense